sormas-ui/src/main/java/de/symeda/sormas/ui/configuration/AbstractConfigurationView.java

```diff
@@ -71,6 +72,11 @@ public void refreshMenu(SubMenu menu, Label infoLabel, Label infoLabelSub, Strin
 					null,
 					false);
 			}
+			menu.addView(
```

Comment:
Move this one up - between outbreaks and area

Reply:
But inside the `INFRASTRUCTURE_VIEW `block?

Reply:
Yes